### PR TITLE
Use the static build of the trustify-ui

### PR DESCRIPTION
**Pull request by chirino**
This should eliminate the NPM builds.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -124,7 +124,7 @@ trustify-module-ui = { path = "modules/ui", default-features = false }
 trustify-module-advisory = { path = "modules/advisory" }
 trustify-module-vulnerability = { path = "modules/vulnerability" }
 trustify-server = { path = "server", default-features = false }
-trustify-ui = { git = "https://github.com/trustification/trustify-ui.git" }
+trustify-ui = { git = "https://github.com/chirino/trustify-ui.git", tag = "static-main" }
```
[the remark this answers is not in the document]
Comment (chirino) [reply Cargo.toml]: didn't know! but I've updated the PR to point at a more permanent location.

**Comment by bobmcwhirter** [comment]
Given that this is "The Jens Situation", I'll leave it to @ctron for review.

**Comment by ctron** [comment]
> Given that this is "The Jens Situation", I'll leave it to @ctron for review.

I always wanted to a have situation named after me :grinning: … now I need to figure out the meaning of it.

**Review by ctron — APPROVED** [review]
❤️ Works on my machine. I like that!

**Pull request merged**